server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -706,6 +706,10 @@ TabletGoalState getGoalState(TabletMetadata tm, MergeInfo mergeInfo) {
         return TabletGoalState.UNASSIGNED;
       }
 
+      if (tm.getOperationId() != null) {
```

Comment:
Might be able to modify ManagerAssignmentIT.test to insert an OPID into tablet metadata and test this

Reply:
done in e2f4500